<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0.0753518, y1:0.08, x2:1, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(0, 255, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="TitreProgramme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-30</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Bold Font</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HandController V.0.6.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonDemarrer">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Démarrer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonParametre">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Paramètres</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonArret">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Arrêter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
